syntax = "proto3";

package array;

service ArrayService {
  // 获取单个640x512的数组
  rpc GetArray(Empty) returns (ArrayResponse);

  // 获取640x512的数组流
  rpc GetArrayStream(Empty) returns (stream ArrayResponse);
}

// 空请求消息
message Empty {}

// 数组响应消息
message ArrayResponse {
  repeated float row = 1; // 存储数组的一行，640个float
}
